st2common/st2common/rbac/types.py

```diff
@@ -196,7 +197,8 @@ def get_permission_name(cls, permission_type):
         :rtype: ``str``
         """
         split = permission_type.split('_')
-        assert len(split) >= 2
+        if not len(split) >= 2:
+            raise ValueError('The length should be greater than or equal to 2.')
```

Comment:
Same issue here - needs a better comparison and a better error message that identifies the root issue.